tests/app/views/test_search_queries.py

```diff
@@ -16,30 +16,6 @@
 
 # Helpers for 'result_fields_check'
 
-def matches(expected_field):
```

Comment:
Ditching these helper definitions altogether seems like a pretty broad interpretation of fixing "do not assign a lambda expression, use a def".  Personally I find e.g. `matches(30)`  in the tests themselves much easier to immediately understand what it is they are looking for than `(30).__eq__`, so I'd be tempted to figure a way to keep these helpers, just without the lambda assignment.

Reply:
For me having `matches(30)` in no way indicated that it's returning a comparison function which was super confusing when I first read it. It's not a very common pattern.
There's no mistaking `__eq__` and `__contains__` for what they are, functions. For me, even if people need to read up what they do it means they understand what the code is doing. That we're providing a comparison function. `matches(30)` as a comparison function feels very arcane. 
I'd expect `lambda i: i == 30` inline maybe. I'm happy to replace it with that if you'd prefer :man_shrugging:

Reply:
or i could redefine them to
```
def get_match_function(match)
    return lambda i: i == match
```

`get_match_function(30)`

Reply:
Nah it's fine as is I think.